swap/src/bin/swap.rs

```diff
@@ -175,8 +177,8 @@ async fn main() -> Result<()> {
             let seller_peer_id = db.get_peer_id(swap_id)?;
 
             let mut swarm = swarm::bob(&seed, seller_peer_id, tor_socks5_port).await?;
-            let bob_peer_id = swarm.local_peer_id();
-            tracing::debug!("Our peer-id: {}", bob_peer_id);
+            let our_peer_id = swarm.local_peer_id();
+            tracing::debug!(peer_id = %our_peer_id, "Our");
```

Comment:
TBH, I find these log messages really weird.

To me, the message should make sense even if the fields wouldn't be there. Can we not say something more general at what just happened in the program where it makes sense to attach the peer-ID to it? Like:

```
tracing::debug!(peer_id = %our_peer_id, "Initializing networking module");
```

Reply:
To be fair, the old log message also wasn't great in that regard :)